docs/openshift.asciidoc

```diff
@@ -142,3 +152,75 @@ Use the following command to get the hosts of each `Route`:
 ----
 oc get route -n elastic
 ----
+
+[float]
+[id="{p}-openshift-apm"]
+=== Deploy an APM Server instance with a route
+
+It is currently not possible to run the APM Server with the `restricted` SCC. But as a workaround you may allow the Pod to run with the default `uid 1000` by assigning it to the `anyuid` SCC:
+
+. Create a service account to run the APM Server
++
+[source,shell]
+----
+oc create serviceaccount apm-server -n elastic
+----
+. Add the APM service account to the `anyuid` SCC
++
+[source,shell]
+----
+oc adm policy add-scc-to-user anyuid -z apm-server -n elastic
+----
++
+[source,shell]
+----
+scc "anyuid" added to: ["system:serviceaccount:elastic:apm-server"]
+----
+. Deploy an APM Server and a route with the following manifest
++
+[source,shell]
+----
+cat <<EOF | oc apply -n elastic -f -
+apiVersion: apm.k8s.elastic.co/v1alpha1
+kind: ApmServer
+metadata:
+  name: apm-server-sample
+spec:
+  version: 7.2.0
+  nodeCount: 1
+  elasticsearchRef:
+    name: "elasticsearch-sample"
+  podTemplate:
+    spec:
+      serviceAccountName: apm-server
+---
+apiVersion: v1
+kind: Route
+metadata:
+  name: apm-server-sample
+spec:
+  #host: apm-server.example.com # override if you don't want to use the host that is automatically generated by OpenShift (<route-name>[-<namespace>].<suffix>)
+  tls:
+    termination: passthrough # the APM Server is the TLS endpoint
+    insecureEdgeTerminationPolicy: Redirect
+  to:
+    kind: Service
+    name: apm-server-sample-apm-http
+EOF
+----
++
+You can check that the Pod of the APM Server is using the correct SCC with the following command:
++
```

Comment:
To check that the Pod of the APM Server is using the correct SCC, use the following command: